pkg/apis/workspaces/v1alpha1/commands.go

```diff
@@ -121,6 +121,14 @@ type ExecCommand struct {
 	// Optional list of environment variables that have to be set
 	// before running the command
 	Env []EnvVar `json:"env,omitempty"`
+
+	// +optional
+	// Whatever the command needs to be restarted when files changed.
```

Comment:
If you want to have your comment nicely rendered as workdown (with new lines and a markdown list) in the generated documentation + Documentation hovers, you could build your comment this way:

```
	// Whatever the command needs to be restarted when files changed.
	// It should be set to "false" if command is capable to do automatic hotreload.
	//
	// Default value:
	//
	// - `true` for group.kind: build, test.
	//
	// - `false` for group.kind: run, debug.
	//
	// - `false` for all commands without group.kind
```

Reply:
This is nice, did not know about this. I've updated the description.